JuliaGPU/session2/Feyaz/qn1.jl

```diff
@@ -0,0 +1,30 @@
+#Try using arr as the 10 digit array, if you're impatient.
+#I tried it twice, and built in sort function was around 67+ ms, 
+# and insertion sort was around 1 ms. I'll put a picture, too.
+using BenchmarkTools
+arr = rand(Int, 1048576)    #2^20=1048576
+#arr = [9,7,8,4,5,3,6,2,1,0]
+len = lastindex(arr)
+
+#Insertion SORT!
+function insort(a)
```

Comment:
```suggestion
function insort!(a)
```
It's a convention in Julia to terminate mutating function with '!'.